src/LaborMarket/LaborMarket.sol

```diff
@@ -231,38 +162,46 @@ contract LaborMarket is
             "LaborMarket::signal: Already signaled."
         );
 
-        // Lock reputation here
```

Comment:
Rather confusing upon the first review due to the lack of apparent enforcement + lack of documentation. Please make sure you're keeping the intended functionality of updated functions up to date please.

Feel as if something is slipping through the cracks here will need to come back to this one as I cannot confirm yet.

Reply:
Yeah my bad I should have documented more but I was not sure if I was completely off base. Next commit will have updated documentation for everything